### ISSUE TITLE
admin dashboard: make the move dataset page more user-friendly

### ISSUE THREAD

**Issue by donsizemore** [IQSS/dataverse#8269]
dashboard-datamove.xhtml requests the DOI of a dataset, and the "identifier" of the host dataverse.

The host "little-d" dataverse is now a collection, and auto-complete seems to accept the alias, which archivists can determine much more easily than the database identifier.

I propose that we `s/Dataverse/Collection/g` and, in the host entry field, make the example `s/Identifier/alias/`

Pull request forthcoming.

**Comment by scolapasta** [comment]
Note, while in the back end (and probably way too many places in the guide) we call it alias, in the front end, we changed it to "identifier":

![image](https://user-images.githubusercontent.com/4268292/144457204-3da41cb7-507a-486f-96a9-b17c59edf371.png)

i.e. when we're talking about db id's we usually say id or database id. Identifer here, I think, was meant to be less confusing than alias and more in line with dataset identifiers (i.e. PIDs).

**Comment by donsizemore** [comment]
@scolapasta okay, but the API still uses a mix of database ID, dataverse alias, and PID, and `/api/dataverses/root` returns `alias:`

I'll close this issue if you'd like, but I still think that "identifier" is ambiguous between DB, PID, and alias.

**Comment by scolapasta** [comment]
I don't think you need to close - I agree with you in part, which is why I mentioned the guides. I just think that, based on the fact that switching that in the UI was an active decision made some time ago, the changes should be the other way from alias -> identifier. Or at least actively decided again that we want this to be referred to as alias (which personally, I am ok with)
